ci/scripts/run_pytest.sh

```diff
@@ -29,8 +29,8 @@ python -m coverage erase
 
 for dir in docs/ ./sub-packages/bionemo-*/; do
     echo "Running pytest in $dir"
-    python -m coverage run --parallel-mode --source sub-packages/ -m pytest -v --nbval-lax $dir
+    python -m coverage run --parallel-mode --source bionemo -m pytest -v --nbval-lax $dir
 done
 
 python -m coverage combine
-python -m coverage report
+python -m coverage report -m
```

Comment:
Is this really the syntax? `-m ... -m`?

Reply:
you're right -- better add a few more -m's for good measure 🤣  

But in all seriousness, yes. Might be less confusing as 
```
python -m coverage report --show-missing
```
but the first invokes coverage (for some reason the `coverage` cli isn't in our path) and the second asks the report to show missing line numbers

Reply:
+1 to use `--show-missing` instead of `-m`